src/app/app-extras.module.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  RestrictedViewModule
+} from './public';
+
+// Specify entry components, module-level providers, etc. here.
+@NgModule({
+  imports: [
+    RestrictedViewModule
```

Comment:
I believe exporting the module is all that's necessary here, no need for the import.